packages/eslint-config-eslint/default.yml

```diff
@@ -1,10 +1,6 @@
 extends:
     - "eslint:recommended"
     - "plugin:node/recommended"
-
-plugins:
-    - "node"
```

Comment:
Why are you removing the Node plugin here? It seems we should specify it if we're going to extend from `"plugin:node/recommended"`.

Reply:
it has been added in `node/recommended`:
https://github.com/mysticatea/eslint-plugin-node/blob/master/lib/configs/recommended.json#L13

Reply:
Oh, weird. I thought the plugins entry was necessary in order to reference the plugin config. If it's not required and everything works fine, we could remove it here.

Reply:
I created another PR(#10685 ), since it seems not related to this PR.